controllers/utils/manifestworkreplicaset.go

```diff
@@ -261,6 +261,29 @@ func getLabelSelectorForPlanItem(
 ) []metav1.LabelSelector {
 	result := []metav1.LabelSelector{}
 	switch planItem.Actions[0] {
+	case ibguv1alpha1.Abort:
+		for _, m := range currentSelector {
+			upgradeCompletedNotExist := m.DeepCopy()
+			if upgradeCompletedNotExist.MatchExpressions == nil {
```

Comment:
Shall we do this outside now that we add filtering almost all the time?

Reply:
won't be simplified that much, since for abort on failure we have two selectors